### DISCUSSION TITLE
Normal, Speed, Voice?

### DISCUSSION THREAD

**Discussion by NaymDev** [Pycord-Development/pycord#2680] [Q&A] [answered]
So would I install speed for the best performance? But why does the normal version exist than? Can I have voice and speed or is that not possible?

**Comment by timmypidashev** [comment, marked as answer]
[speed] adds the following two dependencies, which add cython bindings to aiohttp, the network library used to make requests to discord. This can, in some use cases speed up your client, but is not necessary for small to intermediate bots. As a rule of thumb, If your client is not sharded or receiving 10+ requests per second I wouldn't worry too much about [speed].

https://github.com/Pycord-Development/pycord/blob/f75b7f180ea92010c26191f9d33d4181f67f63da/requirements/speed.txt#L2

[voice] put simply adds voice support, allowing for the bot to connect and play audio in voice channels & stages.

I hope this answers your question :+1:
